### PR TITLE
Better regex for the cookie thing

### PR DESCRIPTION
**Pull request by qwertyquerty**
## About The Pull Request

`^\s*([\[\{\s*]){5,}`

Catches more cases

## Why It's Good For The Game

Crashes bad

## Changelog
:cl:
fix: Better temp fix for crashing bug
/:cl:

### CODE REVIEW

**Comment by JamieD1** [comment]
Might as well wait for byond to fix it instead, they will come up with more combinations

**Comment by ghost** [comment]
> Might as well wait for byond to fix it instead.
and other hilarious jokes you can tell yourself.

**Comment by ike709** [comment]
Probably just going to leave cookie checking disabled and panic bunker until lumlum fixes it.

File: code/modules/goonchat/browserOutput.dm [hunk 1]
```diff
@@ -199,11 +199,9 @@ GLOBAL_DATUM_INIT(iconCache, /savefile, new("tmp/iconCache.sav")) //Cache of ico
...
 		return
-	else
-		return
 
 	if(cookie != "none")
-		var/regex/crashy_thingy = regex("(\\\[ *){5}")
+		var/regex/crashy_thingy = regex("^\\s*(\[\\\[\\{\\}\\\]\]\\s*){5,}")
 		if(crashy_thingy.Find(cookie))
 			message_admins("[key_name(src.owner)] tried to crash the server using at least 5 \"\[\" in a row. Ban them.")
 			log_admin_private("[key_name(owner)] tried to crash the server using at least 5 \"\[\" in a row. Ban them.")
```
Comment (francinum) [inline code/modules/goonchat/browserOutput.dm]:
```suggestion
			message_admins("[key_name(src.owner)] tried to crash the server using malformed cookies. Ban them.")
			log_admin_private("[key_name(owner)] tried to crash the server using malformed cookies. Ban them.")
```

**Pull request closed without merging**